src/app/modules/admin/components/team-cards/team-cards.component.ts

```diff
@@ -0,0 +1,82 @@
+import { Component, OnInit } from '@angular/core';
+
+export interface TeamCard {
+  teamName: string;
```

Comment:
Переименовать параметр, т.к итак понятно, что речь идет об имени команды
```suggestion
export interface TeamCard {
  name: string;
```